AGENTS.md

```diff
@@ -0,0 +1,85 @@
+# Agents.md
+
+This file provides guidance to coding agents when working with code in this repository.
+
+## Essential Commands
+
+**Development:**
+- `pnpm run dev` - Start API server with hot reload on port 5000
+- `pnpm run dev:background` - Start background processor
+- `pnpm run dev:temporal-worker` - Start Temporal worker
+- `pnpm run dev:temporal-server` - Start Temporal server for local development
+
+**Database:**
+- `pnpm run db:migrate:latest` - Apply latest migrations
+- `pnpm run db:migrate:reset` - Drop schema and rerun migrations
+- `pnpm run db:seed:import` - Import seed data for local development
+- `pnpm run db:migrate:make` - Generate new migration based on entity changes
```

Comment:
This comes with nuance that it imports everything retro, maybe just say prefer the create for agent mode? it's good at filling structured stuff anyway

Reply:
I usually find it safer to use make, it's battle tested and deterministic. It's up to the developer to make sure their environment is up to date

Reply:
Just afraid it will prompt ALL changes in a file *(pretty sure this is what happens now right?)